aire/src/main/accordion/accordion.ts

```diff
@@ -0,0 +1,43 @@
+import {bindable, customElement, viewResources} from "aurelia-framework";
+import * as UIkit from "uikit";
+import {Accordion}  from "uikit";
+
+@customElement('aire-accordion')
+@viewResources("./accordion-item")
+export class AireAccordion {
+
+    @bindable
+    animation   : boolean = true;
```

Comment:
use past participle.  Present participles are used for when an action or state is currently active.

Reply:
And just parts of speech are used when specifying a non-boolean or integral state